### PR TITLE
london class 8 -Laleh Koupaei-Html/Css week1

### PR DESCRIPTION
**Pull request by lalehkoupa**
I have finished desktop version and also additional section, now I am working on mobile version that is not finished yet.

**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:Laleh Koupaei
- Your City:London
- Your Slack Name:Laleh(Trainee)

# Homework Details

- Module:Html/Css
- Week:1

# Notes

- What did you find easy?writing Html part was easy for me.

- What did you find hard?implementing the mobile version.

- What do you still not understand?I am working on mobile version and try to understand more concept about media query

- Any other notes?No, thank you

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,85 @@
...
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+        <header class="header">
+            <img src="./img/karma-logo.svg" alt="Karma logo" class="logoImg"/>
+            <nav>
+                <ui class="navList">
+                    <li class="navItem">
+                        <a href="#" class="navLink"><span class="makeBlack">Meet Kamra</span></a>
```
Comment (Sam-Martin) [inline index.html:18]: Typo here

File: index.html [hunk 2]
```diff
@@ -1,19 +1,85 @@
...
+                        <a href="#" class="navLink">Help</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">Login</a>
+                    </li>
+                </ui>
+            </nav>
+            <img class="hamburgerMenu" src="./img/menu-hamburger.svg" alt="menu bar"/>
+        </header>
+        <main>
```
Comment (Sam-Martin) [inline index.html:39]: Good use of main 👍

File: index.html [hunk 3]
```diff
@@ -1,19 +1,85 @@
...
+            <section class="introduction">
+                <h1>Introducing Karma</h1>
+                <p>Bring WiFi with you,everywhere you go.</p>
+                <button class="learnmoreBtn">learn more</button>
+            </section>
+            <section>
+                <div class="advantageTitle">
```
Comment (Sam-Martin) [inline index.html:46]: Is there a better tag to use than a combination of a `<div>` and a `<p>`? What type of text is this?

File: index.html [hunk 4]
```diff
@@ -1,19 +1,85 @@
...
+                    </li>
+                </ui>
+            </nav>
+            <img class="hamburgerMenu" src="./img/menu-hamburger.svg" alt="menu bar"/>
+        </header>
+        <main>
+            <section class="introduction">
+                <h1>Introducing Karma</h1>
+                <p>Bring WiFi with you,everywhere you go.</p>
+                <button class="learnmoreBtn">learn more</button>
```
Comment (Sam-Martin) [inline index.html:43]: Capitalisation

File: index.html [hunk 5]
```diff
@@ -1,19 +1,85 @@
...
+            </section>
+        </main>
+        <footer>
+            <hr>
+            <h3>
+                join us on
+            </h3>
+            <a href="#"><img src="img/twitter-icon.svg" alt="Twitter logo"></a>  
+            <a href="#"><img src="img/facebook-icon.svg" alt="Facebook logo"></a>
+            <a href="#"><img src="img/instagram-icon.svg" alt="Instegram logo"></a>
```
Comment (Sam-Martin) [inline index.html:81]: typo of Instagram

File: index.html [hunk 6]
```diff
@@ -1,19 +1,85 @@
...
+                <div class="getkarmaBackground">
+                </div>
+                <div class="getkarmaTitle">
+                    <p><em>whenever I am,I just don't worry about my connection any more</em></p>
+                    <button class="learnmoreBtn">Get karma today</button>
+                </div>  
+            </section>
+        </main>
+        <footer>
+            <hr>
```
Comment (Sam-Martin) [inline index.html:75]: Is there a way to get the appearance of a line without using an additional element?

File: index.html [hunk 7]
```diff
@@ -1,19 +1,85 @@
...
+            <h3>
+                join us on
+            </h3>
+            <a href="#"><img src="img/twitter-icon.svg" alt="Twitter logo"></a>
```
Comment (Sam-Martin) [inline index.html:79]: Is there a semantic tag that could be used to group these links?

File: css/style.css [hunk 8]
```diff
@@ -5,15 +5,195 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+/*Header
+-----------------------------------------------------*/
+.header{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 10px 50Px ;
```
Comment (Sam-Martin) [inline css/style.css:14]: Sizing should be specified in relative units (`em`/`rem` rather than `px`)

File: css/style.css [hunk 9]
```diff
@@ -5,15 +5,195 @@ body {
...
+    background-image: url('../img/first-background.jpg');
+    background-size: 117%;
+    background-repeat: no-repeat;
+    height: 47vh;
+    color: white;
+    text-align: center;
+    padding:15% 0;
+   }
+
+   .introduction h1{
```
Comment (Sam-Martin) [inline css/style.css:62]: Only the specification inside the `{}` should be indented. `.introduction h1{` itself should be at column 0.

File: css/style.css [hunk 10]
```diff
@@ -5,15 +5,195 @@ body {
...
+.learnmoreBtn{
+    color: white;
+    background-color: rgb(236, 119, 52);
+    border: none;
+    border-radius: 2px;
+    padding: 15px;
+    margin-top: 1.5%;
+}
+
+.introduction button:{
```
Comment (Sam-Martin) [inline css/style.css:84]: Rogue `:` here

File: css/style.css [hunk 11]
```diff
@@ -5,15 +5,195 @@ body {
...
+footer p{
+    color: lightgrey;
+}
+
+footer h3{
+    font-weight: 400;
+    font-size: 1.3rem;
+}
 
+/*media query*/
```
Comment (Sam-Martin) [inline css/style.css:198]: Don't worry about the media query, it's not officially part of the assignment apparently :)

File: css/style.css [hunk 12]
```diff
@@ -5,15 +5,195 @@ body {
...
+    margin: 3rem 0;
+}
+
+.getKarma p{
+    width: 40vW;
+    text-align: center;
+    font-size: 2.5rem;
+    font-weight: 10;
+    color: rgb(71, 67, 67);
+    }
```
Comment (Sam-Martin) [inline css/style.css:154]: Overindentation here!

File: index.html [hunk 13]
```diff
@@ -1,19 +1,85 @@
...
+                    <div class="advantageColumn">
+                        <img src="./img/icon-refill.svg" alt="refill picture"> 
+                        <p>Pay as You Go</p> 
+                    </div>
+                </div>    
+            </section>
+            <section class="getKarma">
+                <div class="getkarmaBackground">
+                </div>
+                <div class="getkarmaTitle">
```
Comment (Sam-Martin) [inline index.html:68]: Is there a better tag you can use to style a title rather than a combination of a `<div>` and `<p>`?

File: css/style.css [hunk 14]
```diff
@@ -5,15 +5,195 @@ body {
...
+    background-color: rgb(236, 119, 52);
+    border: none;
+    border-radius: 2px;
+    padding: 15px;
+    margin-top: 1.5%;
+}
+
+.introduction button:{
+     width: 130px;
+    }
```
Comment (Sam-Martin) [inline css/style.css:86]: Overindented :)

File: css/style.css [hunk 15]
```diff
@@ -5,15 +5,195 @@ body {
...
+    padding:15% 0;
+   }
+
+   .introduction h1{
+       font-size: 3rem;
+       font-weight: lighter;
+
+   }
+   
+   .introduction p{
```
Comment (Sam-Martin) [inline css/style.css:68]: Overindented here too.

File: index.html [hunk 16]
```diff
@@ -1,19 +1,85 @@
...
+        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+        <header class="header">
+            <img src="./img/karma-logo.svg" alt="Karma logo" class="logoImg"/>
+            <nav>
+                <ui class="navList">
```
Comment (Sam-Martin) [inline index.html:16]: This should be `ul` rather than `ui` as it stands for `unordered list` :)

**Review by Sam-Martin — COMMENTED** [review]
Rendered page looks good! Some nitpicks but overall nice work 👍

File: index.html (hunk 2 above)
Comment (lalehkoupa) [reply index.html:39]: Thanks a lot Sam for reviewing my work,I will try to fix all the problems you have mentioned.

File: index.html (hunk 1 above)
Comment (lalehkoupa) [reply index.html:18]: Done!

File: index.html (hunk 4 above)
Comment (lalehkoupa) [reply index.html:43]: Done!

File: index.html (hunk 5 above)
Comment (lalehkoupa) [reply index.html:81]: Done!

File: index.html (hunk 16 above)
Comment (lalehkoupa) [reply index.html:16]: Done!

File: css/style.css (hunk 15 above)
Comment (lalehkoupa) [reply css/style.css:68]: Done!

File: css/style.css (hunk 14 above)
Comment (lalehkoupa) [reply css/style.css:86]: Done!

File: css/style.css (hunk 12 above)
Comment (lalehkoupa) [reply css/style.css:154]: Done!

File: index.html (hunk 3 above)
Comment (lalehkoupa) [reply index.html:46]: Yes I can use <h2>

File: index.html (hunk 6 above)
Comment (lalehkoupa) [reply index.html:75]: Yes I can add border-top to my footer and also give margin to set the border length.

File: css/style.css (hunk 8 above)
Comment (lalehkoupa) [reply css/style.css:14]: Done!

File: css/style.css (hunk 9 above)
Comment (lalehkoupa) [reply css/style.css:62]: Done!

File: index.html (hunk 13 above)
Comment (lalehkoupa) [reply index.html:68]: can I use <section>? @Sam-Martin

File: index.html (hunk 7 above)
Comment (lalehkoupa) [reply index.html:79]: can I use <section> ? @Sam-Martin

**Pull request closed without merging**